sources/packages/backend/apps/api/src/utilities/program-year-utils.ts

```diff
@@ -21,3 +26,18 @@ export function getSupportingUserForm(
       throw new Error(`Unknown supporting user type: ${userType}`);
   }
 }
+
+/**
+ * Check if the offering belong to the program year.
+ * @param newOffering offering details.
+ * @param programYear program year details.
+ * @returns returns true if the offering belong to the program
+ * year.
+ */
+export function offeringBelongToProgramYear(
+  newOffering: EducationProgramOffering,
```

Comment:
I would prefer to have the paramter cleary stating what is required, for instance
`Pick<EducationProgramOffering, "studyStartDate"` or `{studyStartDate: string}`.